.gitignore

```diff
@@ -95,6 +95,8 @@ Thumbs.db
 # intelliJ IDEA project files
 .idea/
 *.iml
+*.iws
+*.ipr
```

Comment:
If it's not related to this PR, can you please create new minor PR for handling it?